server/data-import/get-other-services-csv.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const _ = require('lodash')
+const config = require('config')
+const fs = require('fs')
+const Log = require('log')
+const mongoose = require('mongoose')
+const P = require('bluebird')
+const stringify = require('csv-stringify')
+
+const pregnancyCenterServices = require('../pregnancy-centers/pregnancy-center-services')
+const PregnancyCenterModel = require('../pregnancy-centers/schema/mongoose-schema')
+
+mongoose.Promise = require('bluebird')
+const log = new Log('info')
+
+// TODO: Error handling
+const startDatabase = P.coroutine(function *startDatabase() {
+	yield mongoose.connect(config.mongo.connectionString)
+
+	log.info('Connected to database')
+})
+
+startDatabase()
+
+const mapYesNo = (b) => {
+	if (_.isUndefined(b)) return ''
+	return b ? 'Yes' : 'No'
+}
+
+async function getOtherServicesPregnancyCenters() {
+
+	let data = []
+
+	const services = _.reduce(pregnancyCenterServices, function(obj,service) {
+		obj[service.id] = service.name
+		return obj
+	}, {})
+
+	const columns = _.merge({
+		_id: '_id',
+		prcName: 'Name',
+		note: 'Notes',
+		otherServices: 'Other Services',
+	}, services)
+	
+	const servicesKeys = _.keys(services)
+
+	const pregnancyCenters = await PregnancyCenterModel.find({otherServices:{$ne:null}})
+	for (const pregnancyCenter of pregnancyCenters) {
+		log.info(pregnancyCenter.prcName)
+		const line = [pregnancyCenter._id, pregnancyCenter.prcName, pregnancyCenter.notes || '',
+			pregnancyCenter.otherServices]
+		const servicesValues = _.mapValues(pregnancyCenter.services, mapYesNo)
```

Comment:
No changes necessary, but this line seems like a good candidate for a ramda function! I've been using it more lately and really prefer it over lodash now.